docs/topics/migrations.txt

```diff
@@ -276,6 +276,37 @@ class to make it importable::
 Please refer to the notes about :ref:`historical-models` in migrations to see
 the implications that come along.
 
+.. _initial-property:
+
+Migration initial property
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 1.9
+
+The "initial migrations" for an app are the migrations that create the first
+version of that app's tables. Usually an app will have just one initial
+migration, but in some cases of complex model interdependencies it may have two
+or more.
+
+Initial migrations are marked with an ``initial = True`` class attribute on the
```

Comment:
I believe [class attribute](http://www.diveintopython.net/object_oriented_framework/class_attributes.html) is the correct terminology. Properties are different from attributes.